shared/chat/inbox/index.desktop.js

```diff
@@ -137,20 +139,25 @@ class Inbox extends PureComponent<Props, State> {
       this.props.showSmallTeamsExpandDivider && (
         <BigTeamsDivider toggle={this.props.toggleSmallTeamsExpanded} />
       )
+    const buildTeam = this.props.showBuildATeam && <BuildTeam onBuildTeam={this.props.onBuildTeam} />
     return (
       <ErrorBoundary>
         <div style={_containerStyle}>
-          <ChatFilterRow
-            isLoading={this.props.isLoading}
-            filter={this.props.filter}
-            onNewChat={this._prepareNewChat}
-            onSetFilter={this.props.onSetFilter}
-            hotkeys={isDarwin ? ['command+n', 'command+k'] : ['ctrl+n', 'ctrl+k']}
-            onHotkey={this.props.onHotkey}
-            filterFocusCount={this.props.filterFocusCount}
-            onSelectUp={this.props.onSelectUp}
-            onSelectDown={this.props.onSelectDown}
-          />
+          {this.props.showNewChat ? (
```

Comment:
I think we should change some of this plumbing. We should make a container that makes this decision, maybe ChatInboxHeader (or something), that would be connected and derive showNewChat and would then render one of these 2 components (aka inbox/index.desktop.js would just have <ChatInboxHeader filterFocusCount={} />